zwrap/sample_test.go

```diff
@@ -185,20 +178,21 @@ func TestSamplerCheck(t *testing.T) {
 	assert.Equal(t, expected, sink.Logs(), "Unexpected output when sampling with Check.")
 }
 
-func TestSamplerCheckPanicFatal(t *testing.T) {
-	for _, level := range []zap.Level{zap.FatalLevel, zap.PanicLevel} {
-		sampler, sink := fakeSampler(zap.FatalLevel+1, time.Millisecond, 1, 10, false)
-
-		assert.Nil(t, sampler.Check(zap.DebugLevel, "foo"), "Expected a nil CheckedMessage at disabled log levels.")
-		for i := 0; i < 5; i++ {
-			if cm := sampler.Check(level, "sample"); assert.True(t, cm.OK(), "expected %v level to always be OK", level) {
-				cm.Write(zap.Int("iter", i))
-			}
-		}
-
-		assert.Equal(t, []spy.Log(nil), sink.Logs(), "Unexpected output when sampling with Check.")
-	}
-}
+// TODO: restore this test, now that panic and fatal are actually terminal
```

Comment:
I can test panic easily enough, but fatal would require access to the unexpected exit stubbing; suggestions?